<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>1168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1381</width>
      <height>1121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>31</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TRANSSPACE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добро пожаловать в приложение</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>400</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: yellow;
border-radius: 20px;

</string>
      </property>
      <property name="text">
       <string>Начнём</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>140</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>aakar</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fio">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width: 2px;
border-color:rgb(52, 101, 164);
border-radius:15px;</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="email">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width: 2px;
border-color:rgb(52, 101, 164);
border-radius:15px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width: 2px;
border-color:rgb(52, 101, 164);
border-radius:15px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>430</y>
        <width>621</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style: outset;
border-width: 2px;
border-color:rgb(52, 101, 164);
border-radius:15px;
</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rool_check">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>500</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>500</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Inter','sans-serif'; color:#444444; background-color:rgba(0,0,0,0.058824);&quot;&gt;Я принимаю условия Пользовательского соглашения и даю своё согласие на обработку моей персональной информации на условиях, определенных Политикой конфиденциальности.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="regButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>640</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);
border-style: outset;
border-width: 2px;
border-color:rgb(186, 189, 182);
border-radius:15px;</string>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
     <widget class="QLabel" name="email_status">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>261</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="email_status_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>261</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="password_1_status">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>261</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="password_2_status">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>261</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
